packages/esm-patient-chart-app/src/index.ts

```diff
@@ -83,6 +83,14 @@ function setupOpenMRS() {
           moduleName,
         }),
       },
+      {
+        name: 'mark-alive-button',
+        slot: 'patient-actions-slot',
+        load: getAsyncLifecycle(() => import('./actions-buttons/mark-patient-alive.component'), {
+          featureName: 'patient-actions-slot',
```

Comment:
featureName is actually related to the component, not the slotName. It's good this way too.